github/ci/remote-cluster/templates/mirror-proxy.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: docker-mirror-proxy
+spec:
+  selector:
+    matchLabels:
+      app: docker-mirror-proxy
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: docker-mirror-proxy
+    spec:
+      terminationGracePeriodSeconds: 180
+      serviceAccountName: "caches"
+      containers:
+      - name: mirror-proxy
+        image: fgimenez/docker-registry-proxy:v0.2.0
```

Comment:
In the long run we should push that to kubevirtci org on quay also.

Reply:
makes total sense :+1: